metron-interface/metron-config/src/app/sensors/models/parser-config.model.ts

```diff
@@ -0,0 +1,90 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {FieldTransformer} from '../../model/field-transformer';
+import { ParserModel } from './parser.model';
+import * as cloneDeep from 'clone-deep';
+
+export class ParserConfigModel implements ParserModel {
```

Comment:
Does this class replace SensorParserConfig?  Why not just rename SensorParserConfig to this?  Why would we have 2 classes that are almost the same?

Reply:
@merrimanr this was a rename what was considered as a deletion and a creation by git.

Reply:
I don't see where SensorParserConfig was deleted in this commit.

Reply:
@merrimanr I found it. Like I said it was recognized as a delete/create by git and we added the delete as part of code cleanup later on: https://github.com/apache/metron/pull/1430/commits/7a665a64070b2218492e8c0afe94636046f4815f 
Are you ok with this?

Reply:
I'm sorry but I think the code cleanup changes need to be applied here.

Reply:
I agreed. No problems. I pulled in the code cleanup commit.

Reply:
Thank you, looks good.